classes/command/class-controller.php

```diff
@@ -56,7 +56,37 @@ public function convert_to_gutenberg( $args ) {
 		WP_CLI::log( 'Conversion duration: ' . round( $seconds_elapsed ) . ' seconds' );
 	}
 
-	// Add here new sub-commands e.g. wp p4-blocks new_sub_command.
+	/**
+	 * Sub command that removes duplicate postmeta records
+	 *
+	 * @throws WP_CLI\ExitException The thrown exception.
+	 */
+	public function remove_duplicate_postmeta() {
+
+		$start = microtime( true );
+
+		try {
+			WP_CLI::log( 'Removing duplicate postmeta records...' );
+
+			$instance   = new Remove_Duplicate_Postmeta();
+			$is_deleted = $instance->remove_duplicate();
+
+			if ( $is_deleted ) {
+				WP_CLI::success( 'Removed all duplicate postmeta records' );
```

Comment:
`$is_deleted` will actually contain the number of deleted rows, so you could log this number here too. Maybe you can rename it to `$deleted_rows`.